lib/generators/react_on_rails/templates/base/base/client/webpack.config.js

```diff
@@ -5,9 +5,9 @@
 const webpack = require('webpack');
 const { resolve } = require('path');
 const ManifestPlugin = require('webpack-manifest-plugin');
-const webpackConfigLoader = require('react-on-rails').default.webpackConfigLoader; // eslint-disable-line import/no-unresolved
-
-const { env, paths, publicPath } = webpackConfigLoader(resolve('..', 'config', 'webpack'));
+const webpackConfigLoader = require('react-on-rails/node_package/lib/webpackConfigLoader').default;
+const configPath = resolve('..', 'config', 'webpack');
+const { env, paths, publicPath } = webpackConfigLoader(configPath);
```

Comment:
I'm going to change this around a bit as there seems to be no reason to have the webpack config helper go through babel. Thus, I can move the file so that the 

`require('react-on-rails/node_package/lib/webpackConfigLoader')`

is simpler, like this:

`require('react-on-rails/webpackConfigLoader')`